docs/organizer-guide/meetups/faq-meetups.rst

```diff
@@ -0,0 +1,34 @@
+Meetups FAQ
+===========
+
+The following list of questions and answers cover some issues that you might
+encounter when you want to `start a meetup <http://www.writethedocs.org/organizer-guide/meetups/starting/>`_.
```

Comment:
Might be useful to use the Sphinx doc linking syntax here, instead of pointing at the URL.